docs/reference/settings/security-settings.asciidoc

```diff
@@ -278,10 +278,13 @@ Defaults to `true`.
 The `type` setting must be set to `ldap`. In addition to the 
 <<ref-realm-settings>>, you can specify the following settings: 
 
-`url`:: Specifies one or more LDAP URLs in the format  
-`ldap[s]://<server>:<port>`. Multiple URLs can be defined using a comma 
-separated value or array syntax: `[ "ldaps://server1:636", "ldaps://server2:636" ]`. 
-`ldaps` and `ldap` URL protocols cannot be mixed in the same realm. Required.
+`url`::
+One or more LDAP URLs in the `ldap[s]://<server>:<port>` format. Required.
++
+To provide multiple URLs, use a YAML array (`["ldap://server1:636", "ldap://server2:636"]`)
+or comma-separated string (`"ldap://server1:636, ldap://server2:636"`).
++
+While both are supported, you can't mix the `ldap` and `ldaps` protocols.
```

Comment:
The above is correct.

I do wonder whether the last sentence is too colloquial for non-native speakers, but I'll defer to the the docs team on that.

Reply:
Thanks @tvernum. @debadair  @lcawl let me know what you feel about the sentence @tvernum  is referencing here:

`While both are supported, you can't mix the `ldap` and `ldaps` protocols.`

I think we're within style bounds here, but I'd appreciate any feedback. Thanks as always!